.travis.yml

```diff
@@ -24,29 +25,35 @@ addons:
   apt:
     update: true
     sources:
-      - ubuntu-toolchain-r-test # gcc up to 8.0
+      - ubuntu-toolchain-r-test
       - llvm-toolchain-xenial-8
```

Comment:
shouldn't this be added to only the xenial machines (default is bionic now)

Reply:
Hm, I just didn't like the comments, should they be added back? See https://github.com/ComPWA/ComPWA/pull/312#discussion_r420625184 though.

Reply:
I'm referring to the `llvm-toolchain-xenial-8`. I don't know what the default os for travis is, but our default is set to bionic now. I'm not sure if bionic machines would just skip this package or throw an error. Since we do not have any bionic machines we don't really know what happens (hence maybe also add 1 or 2 bionics)

Reply:
Not sure, perhaps better to move this to a separate issue

Reply:
All right.